draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -45,23 +61,126 @@ export default class Toolbar extends React.Component {
    */
   onOverrideContent = (overrideContent) => {
     this.setState({ overrideContent });
-  }
+  };
 
   onSelectionChanged = () => {
     // need to wait a tick for window.getSelection() to be accurate
     // when focusing editor with already present selection
     setTimeout(() => {
       if (!this.toolbar) return;
+
+      if (typeof this.state.width !== 'number') {
+        this.setState({ width: this.toolbar.offsetWidth });
+      }
+
+      const metrics = {
+        rule: 'left',
+        ruleValue: 0,
+        shift: null,
+        shiftValue: 0
+      };
+      // boundings of the selected text
+      const selectionRect = getVisibleSelectionRect(window);
+      if (!selectionRect) return;
+
       const relativeParent = getRelativeParent(this.toolbar.parentElement);
-      const toolbarHeight = this.toolbar.clientHeight;
+      if (relativeParent) {
+        // avoid false
+        const fontSize = window.getComputedStyle(relativeParent, null).getPropertyValue('font-size');
+        // exit if no selection was made
+        if (selectionRect.left === selectionRect.right && selectionRect.height === parseFloat(fontSize)) {
```

Comment:
wouldn't `if (selectionState.isCollapsed()) {` do the same thing?

Reply:
thanks for the tip